src/main/scala/com/evolutiongaming/prometheus/PrometheusHelper.scala

```diff
@@ -19,15 +28,42 @@ object PrometheusHelper {
 
   implicit class GaugeOps(val gauge: Gauge) extends AnyVal {
 
-    def collect[T](f: => T)(implicit numeric: Numeric[T]): Gauge = {
+    def collect[T: Numeric](f: => T): Gauge = {
       val child = new Gauge.Child() {
-        override def get() = numeric.toDouble(f)
+        override def get(): Double = f.toDouble
       }
       gauge.setChild(child)
     }
   }
 
-  implicit class TemporalOps[A: ObserveDuration](val a: A) {
+  /*
+  TODO: bincompat leftover, remove in 2.x
+
+  Without this magic method MiMa complained:
+  static method TemporalOps(java.lang.Object,com.evolutiongaming.prometheus.ObserveDuration)com.evolutiongaming.prometheus.PrometheusHelper#TemporalOps
+  in class com.evolutiongaming.prometheus.PrometheusHelper does not have a correspondent in current version
+
+  The visibility also has to be public, otherwise scalac does not reliably generate static method for both 2.13 and 3
+   */
+  @deprecated(
+    message = "referencing TemporalOps directly is deprecated, use implicit syntax",
+    since = "1.1.0"
+  )
+  def TemporalOps[A](
```

Comment:
Am I right that the method should not be used anywhere? If so - might be mark it as private?

Reply:
Check out the comment a little above this method:
```
  Without this magic method MiMa complained:
  static method TemporalOps(java.lang.Object,com.evolutiongaming.prometheus.ObserveDuration)com.evolutiongaming.prometheus.PrometheusHelper#TemporalOps
  in class com.evolutiongaming.prometheus.PrometheusHelper does not have a correspondent in current version

  The visibility also has to be public, otherwise scalac does not reliably generate static method for both 2.13 and 3
```